src/util/Injected.js

```diff
@@ -118,6 +119,11 @@ exports.LoadUtils = () => {
 
     };
 
+    window.WWebJS.getUploadLimits = async (mediaType) => {
```

Comment:
We should check if this function check the size of the file after the compressing that whatsapp do or not.
Because if it's after the compressing, the size of the file can be bigger than the limit

Reply:
it returns the value before compression (based on the values i took in my browser)